requester/requester.go

```diff
@@ -158,7 +159,10 @@ func (b *Work) makeRequest(c *http.Client) {
 	if err == nil {
 		size = resp.ContentLength
 		code = resp.StatusCode
-		io.Copy(ioutil.Discard, resp.Body)
+		n, _ := io.Copy(ioutil.Discard, resp.Body)
+		if size < 0 && n > 0 {
```

Comment:
IMHO, checking for `size == -1` would make more sense.

Just like in the `response.go` file: https://github.com/golang/go/blob/master/src/net/http/response.go#L281